viite-UI/src/model/RoadAddressProjectCollection.js

```diff
@@ -142,7 +142,15 @@
         name: data[0].value,
         startDate: data[1].value,
         additionalInfo: data[2].value,
-        roadPartList: dirtyRoadPartList
+        roadPartList: _.map(dirtyRoadPartList, function(part){
```

Comment:
And I think that have two lists is propably useful as well on task VIITE-616, where we need to show 2 different lists to user, and we can filter the dirty list there to show the unsaved road parts